packages/Caching/ValueObject/Storage/FileCacheStorage.php

```diff
@@ -23,70 +21,83 @@ public function __construct(
     ) {
```

Comment:
where is the `$directory` configured which is used? when changing the impl we should add a `v2` or similar so persisted cache information does not conflict with the new impl

Reply:
Is this question still relevant or outdated? The original commit had changed